spec/models/cart_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Cart, type: :model do
+  let(:cart) { Cart.new(nil) }
+
+  it "initializes with an empty contents hash" do
+    expect(cart.contents).to eq({}) 
+  end
+
+  it "can add item ids to itself" do
+    cart.add_item(4)
+    expect(cart.contents).to eq('4' => 1)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.